compiler/src/dmd/hdrgen.d

```diff
@@ -116,6 +116,17 @@ public extern (C++) const(char)* toChars(const Initializer i)
     return buf.extractChars();
 }
 
+public extern (C++) const(char)* toChars(const Type t)
```

Comment:
Add this to the headers. Will also need to be added to `frontend.h` too.

Reply:
ok